addons/project/models/project_milestone.py

```diff
@@ -12,12 +12,13 @@ class ProjectMilestone(models.Model):
     _name = 'project.milestone'
     _description = "Project Milestone"
     _inherit = ['mail.thread']
-    _order = 'deadline, is_reached desc, name'
+    _order = 'sequence'
```

Comment:
We should keep the initial order and add the sequence in first place.

```suggestion
    _order = 'sequence, deadline, is_reached desc, name'
```

Reply:
> _order = 'sequence, deadline, is_reached desc, name'

When the milestone sequences are the same, the project update uses the next order for sorting, but the list view does not. As a result, the mismatch issue occurs again.

Reply:
It is because of that code: 
https://github.com/odoo/odoo/blob/e401171c804b7f931dd98d26a88efdd43d36f070/addons/web/static/src/views/list/list_arch_parser.js#L221-L224

to fix that we have to add the same order in the `default_order` attribute in the list view.

Reply:
> It is because of that code:
> 
> https://github.com/odoo/odoo/blob/e401171c804b7f931dd98d26a88efdd43d36f070/addons/web/static/src/views/list/list_arch_parser.js#L221-L224
> 
> to fix that we have to add the same order in the `default_order` attribute in the list view.

Is it a good idea to use default_order in the list view? That was my question in the morning.

> It is because of that code:
> 
> https://github.com/odoo/odoo/blob/e401171c804b7f931dd98d26a88efdd43d36f070/addons/web/static/src/views/list/list_arch_parser.js#L221-L224
> 
> to fix that we have to add the same order in the `default_order` attribute in the list view.

Okay, This morning, I wanted to confirm this.